releasenotes/notes/mcmt-gate-a201d516f05c7d56.yaml

```
@@ -0,0 +1,36 @@
+---
+features_circuits:
+  - |
+    Added the :class:`.MCMTGate` to represent multi-control multi-target
+    operations in a gate. This gate representation of the existing :class:`.MCMT`
+    circuit allows the compiler to select the best available implementation
+    according to the number of auxiliary qubits present in the circuit.
```

This is super-minor, please feel free to ignore it:

```suggestion
    Added the :class:`.MCMTGate` to represent a multi-control multi-target
    operation as a gate. This gate representation of the existing :class:`.MCMT`
    circuit allows the compiler to select the best available implementation
    according to the number and the state of auxiliary qubits present in the circuit.
```
Do we want to list pending deprecations?

---

I would just list the deprecations once they are not pending anymore 🙂 (I think that's what we typically have as well 🙂)